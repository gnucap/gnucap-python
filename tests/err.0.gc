options trace
attach public c_python.so
loadpy parse_err.py

end
