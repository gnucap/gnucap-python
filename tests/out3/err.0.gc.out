Gnucap : The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2013, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
main version
core-lib version
default plugins
>>>>>attach public c_python.so 
>>>>>loadpy parse_err.py 
>>>>>end 
